aa_stripe/models.py

```diff
@@ -125,6 +125,52 @@ def create_at_stripe(self):
         self.save()
         return self
 
+    def update_at_stripe(self, stripe_token, set_default):
+        is_default = self.customer.default_card.pk is self.pk
+        set_default = is_default if set_default is None else set_default
+
+        if is_default and not set_default:
```

Comment:
Why are you trying to "unset" default? The whole point of having "set_default" is that when it is chosen, you make given card default when it is false/not present you just do nothing.

Reply:
I need to update the card anyway from the api, and when it is default and is the only card then Stripe won't allow unsetting the default card. 
This is for the occasion when someone will try to unset the default card when it is the only card. 
But I can just ignore that someone wants to unset the only card from being the default card.

Reply:
But why do you even want to have such action as "unset default"? Set default is all you need.

Reply:
Ok